### ISSUE TITLE
Color output displays escape codes

### ISSUE THREAD

**Issue by cbenz** [fiatjaf/jiq#6]
Hi, I really like `jiq`!

Is there a recommended way to do syntax highlighting with `jiq`?

When I use the `--color-output` of `jq` with `jiq` like this:

```sh
echo "[1,2,3]" | jiq --color-output
```

I see escape codes:

```sh
[Filter]> .
 [1;39m[
   [0;39m1 [0m [1;39m,
   [0;39m2 [0m [1;39m,
   [0;39m3 [0m [1;39m
 [1;39m] [0m
```

Should escape codes be supported at the `terminal.go` level?

**Comment by fiatjaf** [comment]
Yes, they should. But I don't know how to do it. terminal.go uses https://github.com/nsf/termbox-go, which does support colors, but not shell escape codes.

On jid there's [a commit that implements colors](https://github.com/simeji/jid/commit/c1280348b596bdfb1e20b149d1d8c84a2c63c132), but that's independent from jq, a custom way of colouring. I don't know if it makes sense to implement that here, but if you want to try and see if it fits well that would be good.

Or, better, you could draw inspiration from that to learn how to implement ANSI escape codes in termbox.

Another alternative, perhaps the best one, is to replace termbox with some other library that already supports these escape codes out of the box.
